src/__Private/LSPImpl/CodeActionCommand.hack

```diff
@@ -107,7 +107,7 @@ final class CodeActionCommand extends LSPLib\CodeActionCommand {
       if ($code !== ($diagnostic['code'] ?? null)) {
         continue;
       }
-      if ($error->getPosition() !== $pos) {
+      if ($error->getRange() |> $$ is null || $$[0] !== $pos) {
```

Comment:
Neat trick :) though could you change to this for readability?

```hack
$error_position = $error->getRange()[0] ?? null;
if ($error_position !== pos) {
```

The addition of 'is null' appears to be a behavior change here too?

`$foo['bar']['baz'] ?? $x` will evaluate to $x if any of the intermediates in the expression are unset or null.